### PR TITLE
feat(fe): connect admin problem file upload API

### LINKED ISSUE #660: group admin problem upload by file
### Describe the problem and solution

파일로 문제를 업로드하는 기능을 구현한다

### Validations

- [X] Follow our [Code of Conduct](https://github.com/skkuding/next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [X] Check that there isn't already an issue

### PR DESCRIPTION
**Pull request by eunnbi**
### Description

- Admin 문제 파일 업로드 API 연결
- 배포된 개발 서버에 요청하면 500 반환
- 백엔드 로컬 서버에 요청하면 200 반환

close #660

### Additional context

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: frontend/src/admin/pages/[groupId]/problem/index.vue [hunk 1]
```diff
@@ -6,28 +6,67 @@ import Dialog from '@/common/components/Molecule/Dialog.vue'
...
     })
   } else {
-    dialog.success({
-      title: 'Success',
-      content: 'Successfully Uploaded',
-      yes: 'OK'
-    })
+    const operations = {
+      query:
+        'mutation($groupId: Float!, $input: UploadFileInput!) { uploadProblems(groupId: $groupId, input: $input){id createdById groupId title description template languages difficulty}}',
```
Comment (dotoleeoak) [inline frontend/src/admin/pages/[groupId]/problem/index.vue:33]: 여기 gql tag 추가해주세요!

Comment (eunnbi) [reply frontend/src/admin/pages/[groupId]/problem/index.vue:33]: `gql` 태그 추가하면 400 에러 반환해요 😢

Comment (eunnbi) [reply frontend/src/admin/pages/[groupId]/problem/index.vue:33]: `JSON.stringfy()` 과정에서 문제가 발생하는 것 같아요

Comment (eunnbi) [reply frontend/src/admin/pages/[groupId]/problem/index.vue:33]:
```
operations: {"query":{"kind":"Document","definitions":[{"kind":"OperationDefinition","operation":"mutation","variableDefinitions":[{"kind":"VariableDefinition","variable":{"kind":"Variable","name":{"kind":"Name","value":"groupId"}},"type":{"kind":"NonNullType","type":{"kind":"NamedType","name":{"kind":"Name","value":"Float"}}},"directives":[]},{"kind":"VariableDefinition","variable":{"kind":"Variable","name":{"kind":"Name","value":"input"}},"type":{"kind":"NonNullType","type":{"kind":"NamedType","name":{"kind":"Name","value":"UploadFileInput"}}},"directives":[]}],"directives":[],"selectionSet":{"kind":"SelectionSet","selections":[{"kind":"Field","name":{"kind":"Name","value":"uploadProblems"},"arguments":[{"kind":"Argument","name":{"kind":"Name","value":"groupId"},"value":{"kind":"Variable","name":{"kind":"Name","value":"groupId"}}},{"kind":"Argument","name":{"kind":"Name","value":"input"},"value":{"kind":"Variable","name":{"kind":"Name","value":"input"}}}],"directives":[],"selectionSet":{"kind":"SelectionSet","selections":[{"kind":"Field","name":{"kind":"Name","value":"id"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"createdById"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"groupId"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"title"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"description"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"template"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"languages"},"arguments":[],"directives":[]},{"kind":"Field","name":{"kind":"Name","value":"difficulty"},"arguments":[],"directives":[]}]}}]}}],"loc":{"start":0,"end":321}},"variables":{"groupId":2,"input":{"file":null}}}
```

operation이 이렇게 전달돼요!

Comment (eunnbi) [reply frontend/src/admin/pages/[groupId]/problem/index.vue:33]:
```
operations: {"query":"mutation($groupId: Float!, $input: UploadFileInput!) { uploadProblems(groupId: $groupId, input: $input){id createdById groupId title description template languages difficulty}}","variables":{"groupId":2,"input":{"file":null}}}
```

`gql` 태그 안 했을 때

**Review by dotoleeoak — APPROVED** [review]
수고했어요~

**Pull request merged**